### ISSUE TITLE
Show numbers when CTRL is pressed

### ISSUE THREAD

**Issue by DutchPete** [oxdc/obsidian-vertical-tabs#91]
### Enhancement Checklist

- [X] I have searched existing issues and this is not a duplicate
- [X] I am reporting a simple enhancement or improvement within Vertical Tabs, not requesting new features or addressing compatibility issues with other plugins
- [X] I understand that proposals for new features should be discussed in the ‘Discussions’ tab
- [X] I understand that compatibility issues with other plugins should be posted as a ‘Compatibility’ issue

### Enhancement Type

Usability

### Enhancement Description

1 of the ways the tabs in the vertical tab pane can be accessed is by pressing the <kbd>CTRL</kbd> key plus the number of the tab's position. Currently, the tab's position requires one to count from the top down.

### Motivation

It would help the workflow if when pressing one sees all tabs' position numbers, which avoids having to do a count.

### Alternatives (Optional)

N/A

### Additional Information (Optional)

N/A

**Comment by oxdc** [comment]
Hi @DutchPete ! If I'm correct, what you proposed is to allow users to open a tab by pressing CTRL + its index. I'm wondering if this functionality is already implemented in Obsidian? And what if there are more than 10 tabs open?

**Comment by DutchPete** [comment]
Yes, that is what I am proposing. I am not aware Obsidian already has it, I'll check again.

Regarding more than 10 tabs, I imagine the implementation is like a browser: CTRL+9 opens the last  tab. For tabs 10 and later you will have to click on them.

**Comment by oxdc** [comment]
Hi @DutchPete ! Could you please review the following demo video? Is this what you want?

https://github.com/user-attachments/assets/4b3dd2a0-326a-43e9-89cc-52090d910295

**Comment by DutchPete** [comment]
Exactly!!
So, Obsidian has the feature to go to tab 3 by typing CTRL/CMD+3, but it does not show the numbers when you just press CTRL+CMD. Your plug-in will fill that gap.

BTW, which theme are you using in that demo?

**Comment by oxdc** [comment]
> BTW, which theme are you using in that demo?

It's [Primary for Obsidian](https://github.com/primary-theme/obsidian).

**Comment by DutchPete** [comment]
Thank you.

**Comment by oxdc** [comment]
> And what if there are more than 10 tabs open?
>
> Regarding more than 10 tabs, I imagine the implementation is like a browser: CTRL+9 opens the last tab. For tabs 10 and later you will have to click on them.

Hi @DutchPete ! Here's a better solution:

https://github.com/user-attachments/assets/590eb908-64c7-4612-91ee-c4b53224e9ad

**Comment by DutchPete** [comment]
Excellent!! That is a clever way of dealing with an infinite number of tabs because <kbd>CTRL</kbd> + <kbd>➙</kbd> resets the counter to 1 for the next 9 tabs. Well found.

I wonder if you could sell the idea to Google and Mozilla.

**Comment by DutchPete** [comment]
Huh, closed? Will your better solution be implemented?

**Comment by oxdc** [comment]
Hi @DutchPete ! I just finished testing. It will be available in the next release.

**Comment by DutchPete** [comment]
OK, thank you.
